src/olympia/conf/dev/settings.py

```diff
@@ -17,11 +17,12 @@
 API_THROTTLING = True
 
 DOMAIN = env('DOMAIN', default='addons-dev.allizom.org')
+SERVICES_DOMAIN = env('SERVICES_DOMAIN', default='services.addons-dev.allizom.org')
```

Comment:
Are we actually setting environment variables for "SERVICES_DOMAIN" and for "SERVICES_URL"

I see that before we read the SERVICES_URL directly, so assuming it is set, but just incorrect, per your description. But are we also setting the `SERVICES_DOMAIN` env or will we always use the default?

Is the SERVICES_DOMAIN always just `services.{DOMAIN}`, if so why not just set that for the SERVICES_URL since it is already being formatted anyway.

Are we actually using the SERVICES_DOMAIN setting anywhere? I ask because it did not exist in the file before? If it is only used in order to be added to SERVICES_URL and given my previous statement, wouldn't it be cleaner to just set

```python
SERVICES_URL = f"https://services.{DOMAIN}"
```

Or is domain also incorrect based on the environment variables?

Finally, why is the SERVICES_URL incorrect in the environment variables? Could we fix the problem there and remove the need for this PR entirely?

Reply:
- Currently, we are only passing `SERVICES_URL` environment variable, not `SERVICES_DOMAIN`. As mentioned below I'm introducing `SERVICES_DOMAIN` as an environment variable and removing the support for `SERVICES_URL`.

- `SERVICES_URL` was/is correct. The problem was purely `SERVICES_DOMAIN`, which was only set in `lib/settings_base.py` without being based on an environment variable at all as explained in the PR description.

- As I said in the description, both settings are barely used, the only real impact is for the caching duration change which does require `SERVICES_DOMAIN`.

I could change to `SERVICES_DOMAIN = env('SERVICES_DOMAIN', default=f'services.{DOMAIN}')` if you prefer.